instrumentation/kafka/README.md

```diff
@@ -0,0 +1,7 @@
+# Settings for the Kafka instrumentation
+
+| System property                                           | Type    | Default | Description                                                                                                                |
+|-----------------------------------------------------------| ------- |---------|----------------------------------------------------------------------------------------------------------------------------|
+| `otel.instrumentation.kafka.experimental-span-attributes` | Boolean | `false` | Enable the capture of experimental span attributes.                                                                        |
+| `otel.instrumentation.kafka.client-propagation.enabled`   | Boolean | `true`  | Enable context propagation for kafka message producer.                                                                     |
+| `otel.instrumentation.kafka.metric-reporter.enabled`      | Boolean | `true`  | Enable kafka consumer and producer metrics. Deprecated, disable instrumentation with name `kafka-clients-metrics` instead. |
```

Comment:
Can the deprecated property be highlighted or emphasized in some way? Stating "Deprecated" in the description isn't super noticeable